Experimental/MultiplayerUseCases/Packages/manifest.json

```
@@ -0,0 +1,52 @@
+{
+  "dependencies": {
+    "com.unity.ai.navigation": "1.1.4",
+    "com.unity.feature.development": "1.0.1",
+    "com.unity.ide.rider": "3.0.24",
+    "com.unity.ide.visualstudio": "2.0.18",
+    "com.unity.ide.vscode": "1.2.5",
+    "com.unity.inputsystem": "1.7.0",
+    "com.unity.learn.iet-framework": "3.1.3",
+    "com.unity.learn.iet-framework.authoring": "1.2.2",
+    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop",
```

To match the other bitesize samples, let's target the latestz version of the Samples Utilities Package, so:
```suggestion
    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#v2.3.0",
```